integration/e2e/instantiation_policy_test.go

```diff
@@ -51,27 +53,41 @@ var _ = Describe("InstantiationPolicy", func() {
 	})
 
 	AfterEach(func() {
-		if process != nil {
-			process.Signal(syscall.SIGTERM)
-			Eventually(process.Wait(), network.EventuallyTimeout).Should(Receive())
+		if ordererProcess != nil {
+			ordererProcess.Signal(syscall.SIGTERM)
+			Eventually(ordererProcess.Wait(), network.EventuallyTimeout).Should(Receive())
+		}
+
+		if peerProcess != nil {
+			peerProcess.Signal(syscall.SIGTERM)
+			Eventually(peerProcess.Wait(), network.EventuallyTimeout).Should(Receive())
 		}
+
 		if network != nil {
 			network.Cleanup()
 		}
+
 		os.RemoveAll(testDir)
 	})
 
 	Describe("single node etcdraft network with single peer", func() {
 		BeforeEach(func() {
-			config := nwo.MinimalRaft()
-			config.Profiles[1].Organizations = []string{"Org1", "Org2"}
+			config := nwo.MinimalRaftNoSysChan()
+			config.Profiles[0].Organizations = []string{"Org1", "Org2"}
 			network = nwo.New(config, testDir, client, StartPort(), components)
 			network.GenerateConfigTree()
 			network.Bootstrap()
 
-			networkRunner := network.NetworkGroupRunner()
-			process = ifrit.Invoke(networkRunner)
-			Eventually(process.Ready(), network.EventuallyTimeout).Should(BeClosed())
+			// Start all the fabric processes
+			ordererRunner, ordererProcess, peerProcess = startNetwork(network, "orderer")
+
+			//ordererRunner = network.OrdererRunner(network.Orderer("orderer"))
```

Comment:
Let's try to avoid merging commented out code

Reply:
done